### ISSUE TITLE
Clicking a link in the new windows taskbar news popup causes this

### ISSUE THREAD

**Issue by adoublearonn** [spikespaz/search-deflector#142]
> **IMPORTANT:** READ EVERYTHING!!! That includes the wiki. Make sure you've skimmed the wiki before making a bug report.
> https://github.com/spikespaz/search-deflector/wiki
>
> This issue should be properly formatted in markdown. If you don't know what that is, read the following website before continuing:
> https://guides.github.com/features/mastering-markdown/
>
> Most of the text in this issue should be replaced with your own or deleted. Do not delete section headers unless the information they would contain is irrelevant or inapplicable.

## Bug Description

If you have any other details, replace this line, otherwise delete it.

**File:** `source\deflector.d`

**Line:** 53

**Message:**

```
There was an error deflecting your search. Passed URI is:	microsoft-edge:https://www.msn.com/en-us/news/world/biden-s-climate-summit-sets-up-a-bigger-test-of-american-power/ar-BB1fZg7P?ocid=winp1taskbar
```

## How to Reproduce

**Query string:**
```

```

**Query or URL:**
```

```

**Steps to reproduce the behavior:**

## Environmental Details

**Browser**: Google Chrome

**Search Engine:** UNKNOWN STRING

**Custom browser path:**
```
C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
```

**Custom search engine URL:**
```
google.com/search?q={{query}}
```

| Windows Version | `2009` |
| --- | --- |
| Build Number | `21364` |
| Edition | `Enterprise` |
| Insiders Preview |  |

**Comment by MaddyGuthridge** [comment]
Almos definitely an issue where it is expecting a bing URL and panics when it gets given an MSN news one

**Comment by computechnopro** [comment]
> Almos definitely an issue where it is expecting a bing URL and panics when it gets given an MSN news one

Exactly!![bitmoji](https://sdk.bitmoji.com/render/panel/fa95e632-d68f-4e15-81a3-0924a6f41992-137137fb-82d4-467d-bc09-28904354ac1e-v1.png?transparent=1&palette=1&width=246)
